src/components/timeline/Row.svelte

```diff
@@ -569,7 +680,7 @@
       >;
       const text = overlaySvgSelection.select('.activity-drag-guide text');
       const rectWidth = rect.node()?.getBBox()?.width ?? 0;
-      const rectRight = offsetX + rectWidth ?? 0;
```

Comment:
Reason for removing default?

Reply:
It got flagged by VSCode as unnecessary as none of the values would ever be undefined, just zero. So this was unnecessary.